src/clj/game/cards/identities.clj

```diff
@@ -528,6 +528,25 @@
                                          :effect (effect (add-prop target :advance-counter 4 {:placed true}))} card nil)))
                                 (update! state side (assoc (get-card state card) :biotech-used true))))}]}
 
+   "Kabonesa Wu: Netspace Thrillseeker"
+   {:abilities [{:label "[:click] Install a non-virus program from your stack, lowering the cost by 1 [Credit]"
+                 :cost [:click 1]
+                 :prompt "Choose a program"
+                 :choices (req (cancellable
+                                (filter #(and (is-type? % "Program")
+                                              (not (has-subtype? % "Virus")))
+                                        (:deck runner))))
+                 :msg (str"install a non-virus program from their stack, lowering the cost by 1 [Credit]")
```

Comment:
Can you put a space between `str` and `"`? It seems to throw off GitHub's syntax highlighting, even if it works in Clojure.